### PR TITLE
cb - Display images

### LINKED ISSUE #76: Display cover images for entries in user list
Acceptance criteria:
- [x] Each row in the user list includes the cover image from the Anilist query.
- [x] A placeholder image should be displayed for an anime with no cover image (this should be satisifed by the first critierion because Anilist already sets this to a placeholder cover image).
- [x] Given a populated anime list (either search results or user list), all cover images should be of the same height and width when displayed.

### PR DESCRIPTION
**Pull request by christopherbarnett**
- In this PR, cover images are successfully displayed for each anime on the user's list with a standardized height (aspect ratio not changed for individual images though).
- I added an image entry for the columns title field.
- Because Anilist already includes a placeholder image for each anime without an official "cover image," the code for displaying the image will remain the same because it comes from the same field from the anilist query.
- I also quickly standardized the height for the search results. Apparently, not all the image heights were the same before, we just happened to look up anime with cover images of the same or similar height lol.
- Acceptance criteria are all passing.

userListCoverImages

Closes #76

### CODE REVIEW

File: client/src/components/list/UserListRow.tsx [hunk 1]
```diff
@@ -26,6 +26,7 @@ const UserListRow: React.FC<UserListRowProps> = ({ entryData }) => {
 
   return (
     <Tr>
+      <Td><Image src={entryData.coverImage} height="100"/></Td>
```
Comment (jvoucsb) [inline client/src/components/list/UserListRow.tsx:29]: Have a set width and an objectFit "cover". Some cover images are wider.

File: client/src/components/result/SearchResult.tsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ const SearchResult: React.FC<SearchResultProps> = ({ anime }) => {
   }
   return (
     <HStack>
-      <Image src={anime.coverImage.medium} />
+      <Image src={anime.coverImage.medium} height="150"/>
```
Comment (jvoucsb) [inline client/src/components/result/SearchResult.tsx:26]: Same thing here

**Review by jvoucsb — CHANGES_REQUESTED** [review]

**Review by jvoucsb — APPROVED** [review]
good good excellent good

**Review by AHuangHe — APPROVED** [review]
LGTM!

**Pull request merged**